src/EnergyPlus/ScheduleManager.cc

```diff
@@ -490,7 +490,7 @@ namespace ScheduleManager {
                                 ShowSevereError(state, error);
                             }
                         }
-                        ShowContinueError(state, fmt::format("Error Occurred in {}", state.files.TempFullFilePath.filePath.string()));
+                        ShowContinueError(state, fmt::format("Error Occurred in {}", state.files.TempFullFilePath.filePath));
```

Comment:
use the new fmt::formatter<fs::path> everywhere, so we don't call fs::path::string()